inc/classes/preload/class-sitemap.php

```diff
@@ -211,54 +212,100 @@ public function process_sitemap( $sitemap_url, $urls = [] ) {
 			$this->sitemap_error = true;
 
 			set_transient( 'rocket_preload_errors', $errors );
-			return [];
+			return $urls;
 		}
 
 		$url_count        = count( $xml->url );
 		$sitemap_children = count( $xml->sitemap );
 
 		if ( $url_count > 0 ) {
+			$mobile_preload = $this->preload_process->is_mobile_preload_enabled();
+			$mobile_suffix  = '##wpm-mobile##';
```

Comment:
I'm thinking you could define a class constant in the parent class for the mobile suffix, to improve maintainability